sap-hana-mixin/.lint

```diff
@@ -0,0 +1,14 @@
+exclusions:
+  template-job-rule:
+    reason: "Prometheus datasource variable is being named as prometheus_datasource now while linter expects 'datasource'"
+  panel-datasource-rule:
+    reason: "Loki datasource variable is being named as loki_datasource now while linter expects 'datasource'"
+  template-datasource-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  template-instance-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  target-instance-rule: 
+    reason: "SAP HANA uses the label 'host' to identify separate instances"
+  panel-units-rule:
+    reason: "Custom units are used for better user experience in these panels"
+    entries:
```

Comment:
@kuiperda  did you actually run the linter? i notice there are no entries here? or can you delete this rule?

Reply:
Yeah, I noticed that you can run it with no entries listed, Keith had that for redis-enterprise. But deleting the rule gives lint errors.